pyproject.toml

```
@@ -0,0 +1,84 @@
+[build-system]
+requires = ["setuptools", "pip >= 10"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pysatNASA"
+version = "0.0.5"
+description = "pysat support for NASA Instruments"
+readme = "README.md"
+requires-python = ">=3.8"
```

```suggestion
requires-python = ">=3.6"
```
Otherwise this will fail for the operational system.